aiida/cmdline/commands/cmd_code.py

```diff
@@ -174,28 +173,24 @@ def show(code, verbose):
     table.append(['Prepend text', code.get_prepend_text()])
     table.append(['Append text', code.get_append_text()])
 
-    if verbose:
+    if is_verbose():
         table.append(['Calculations', len(code.get_outgoing().all())])
 
-    click.echo(tabulate.tabulate(table))
+    echo.echo(tabulate.tabulate(table))
 
 
 @verdi_code.command()
 @arguments.CODES()
-@options.VERBOSE()
 @options.DRY_RUN()
 @options.FORCE()
 @with_dbenv()
-def delete(codes, verbose, dry_run, force):
+def delete(codes, dry_run, force):
     """Delete a code.
 
     Note that codes are part of the data provenance, and deleting a code will delete all calculations using it.
     """
     from aiida.common.log import override_log_formatter_context
```

Comment:
Do we still need to use this here?

Reply:
I don't think so no, since the logging now goes through the echo functions which essentially do this formatting. Shall I also remove the utility itself, since its only use is in the cmdline commands?

Reply:
Quite proably yeh, again it would be good to run some of these commands once you are done and print the terminal output here, so we can check they are still looking pretty 😄

Reply:
Uhm, there is still a problem. Even though the logging level of the child loggers is properly set on the `AIIDA_LOGGER` and they inherit it, they still have the default formatting. Removing these overrides will therefore return the "incorrectly" formatted log messages. I think we should still remove them, but like the log level, also change the formatter for the `AIIDA_LOGGER` and all its children, globally.

Reply:
OK, commit 6340311 implements a solution for this problem by configuring a custom formatter for all loggers when the CLI is invoked.